### ISSUE TITLE
Connecting/disconnecting a DualShock 4 controller crashes Encore

### ISSUE THREAD

**Issue by BraqueDown** [Encore-Developers/Encore-Raylib#4]
Windows 10 x64, Nightly (a6c290c x64), also affects v0.1.3.

Does not occur with my XB1 controller. Maybe has to do with the audio device the DS4 creates?

**Comment by NarrikSynthfox** [comment]
Yeah that is probably the case is the audio device. I'll see if I can set it to where it uses a set audio device always and doesnt try to change it

**Comment by BraqueDown** [comment]
No longer seems to occur in 83c268e.

**Comment by 3drosalia** [comment]
Was about to ask about this, nice to see it works properly again! :3
